verticapy/sql/flex.py

```diff
@@ -100,6 +120,22 @@ def compute_vmap_keys(
     List of tuples
         List of virtual column names and their respective
         frequencies.
+
+    Examples
+    --------
+    .. code-block:: python
+
+        import verticapy as vp
+        from verticapy.sql import compute_vmap_keys
+
+        vp.read_json("laliga/*.json", table_name = "laliga", materialize = False)
```

Comment:
the general idea for the codeblock is that the users can directly copy the code and execute. We should try to refrain from  reading files in the user directory like vp.read_json("laliga/*.json", table_name = "laliga", materialize = False). Instead can we do this using load_laliga?

Reply:
So when I use load_laliga, I'm getting an error that states "public.laliga" is not a flex table

Reply:
Probably missing something obvious, but I can't get this to work w/o either using sql magic or creating a JSON file and running read_json. If there's an easier way just comment it here, and I'll switch it over.

Reply:
I also couldn't think of an easier way. Below is what I got. But we want to avoid creating and referencing files. @oualib  any other idea? Can we directly consume the json string to a vDataFrame?

```
import json
data = {
    "column1": {
        "subcolumn1A": "value1A",
        "subcolumn1B": "value1B"
        },
    "column2": {
        "subcolumn2A": "value2A",
        "subcolumn2B": "value2B"
        }
    }   

json_string = json.dumps(data, indent=4)

with open("nested_columns.json", "w") as json_file:
    json_file.write(str(json_string))

vp.create_schema("temp")
vdf = vp.read_json("nested_columns.json", schema = "temp", table_name = "test", flatten_maps = False)
print(compute_vmap_keys(expr = "temp.test", vmap_col = "column1"))
vp.drop("temp.test")
```